great_expectations/rule_based_profiler/types/data_assistant_result/plot_result.py

```diff
@@ -1,9 +1,15 @@
 from dataclasses import dataclass
+from enum import Enum
 from typing import List
 
 import altair as alt
 
 
+class PlotMode(Enum):
+    DESCRIPTIVE = "descriptive"
+    PRESCRIPTIVE = "prescriptive"
+
```

Comment:
Opted for an enum instead of a boolean value - any confusion around this should be internally contained because we're not exposing this object to the user.